modules/oe_theme_content_tender/oe_theme_content_tender.module

```diff
@@ -44,6 +64,36 @@ function oe_theme_content_tender_node_view_alter(array &$build, EntityInterface
  * Implements hook_preprocess_HOOK().
  */
 function oe_theme_content_tender_preprocess_field__node__oe_tender_deadline__full(&$variables, $hook) {
+  oe_theme_content_tender_strike_deadline($variables);
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_content_tender_preprocess_field__node__oe_tender_deadline__teaser(&$variables, $hook) {
+  oe_theme_content_tender_strike_deadline($variables);
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_content_tender_preprocess_field__node__oe_departments__teaser(&$variables, $hook) {
+  if (count($variables['items']) <= 1) {
+    return;
+  }
+  $departments = array_shift($variables['items']);
+  foreach ($variables['items'] as $value) {
+    $departments['content']['#plain_text'] .= " | " . $value['content']['#plain_text'];
+  }
+  $variables['items'] = [$departments];
```

Comment:
Use this template suggestion to style this and remove this preprocess completely.

```
   * field--node--oe-departments--oe-tender--teaser.html.twig
```

We already do the same here: `field--node--oe-departments--oe-tender--full.html.twig`

Reply:
Updated